sqladmin/forms.py

```diff
@@ -23,37 +35,63 @@
 )
 from wtforms.fields.core import UnboundField
 
+from sqladmin.exceptions import NoConverterFound
 from sqladmin.fields import JSONField, QuerySelectField, QuerySelectMultipleField
 
 
+class Validator(Protocol):
+    def __init__(self, *args: Any, **kwargs: Any) -> None:
+        ...  # pragma: no cover
+
+    def __call__(self, form: Form, field: Field) -> None:
+        ...  # pragma: no cover
+
+
+class ConverterCallback(Protocol):
+    def __call__(
+        self,
+        model: type,
+        prop: Union[ColumnProperty, RelationshipProperty],
+        kwargs: Dict[str, Any],
+    ) -> UnboundField:
```

Comment:
I will need to read the mypy docs again about this. I think this PR needs to be rebased.

Reply:
I rebased the PR.